app/views/organizations/show.html.erb

```diff
@@ -23,9 +23,17 @@
   </div>
   <div class="box-body">
     <div class="col-md-4">
-      <h4>Contact Info</h4>
-      <p><%= fa_icon "envelope" %> <%= link_to @organization.email, "mailto:#{@organization.email}" %></p>
-      <address><%= fa_icon "map-marker" %> <%= @organization.address %></address>
+      <div>
+        <h4>Contact Info</h4>
+        <p><%= fa_icon "envelope" %> <%= link_to @organization.email, "mailto:#{@organization.email}" %></p>
+        <address><%= fa_icon "map-marker" %> <%= @organization.address %></address>
+      </div>
+      <div>
+        <h4>Reminder day</h4>
+        <p><%= fa_icon "calendar" %> Every day <%= @organization.reminder_date %></p>
+        <h4>Deadline day</h4>
+        <p><%= fa_icon "calendar" %> Every day <%= @organization.deadline_date %></p>
```

Comment:
This is a minor issue, but could you change the langauge to read: 
```suggestion
       <p><%= fa_icon "calendar" %> The <%= @organization.deadline_date + ordinal(@organization.deadline_date) %> of each month</p>
```
For both items, not just that one.

Reply:
This sounds great. I'll apply these changes.